### PR TITLE
Yosys plugins symbiflow

### PR DESCRIPTION
**Pull request by mtdudek**
Adds package with symbiflow-plugins for yosys.
There is package very similar to this one, but that one uses sybmiflow-yosys and not main stream one.

### CODE REVIEW

File: .github/workflows/build-packages.yml [hunk 1]
```diff
@@ -421,7 +434,7 @@ jobs:
     env:
       PACKAGE: "xilinx/vivado"
       OS_NAME: "linux"
-      VERSIONS: ${{ format('{0}{1}', github.workspace, '/xilinx/vivado/versions') }} 
+      VERSIONS: ${{ format('{0}{1}', github.workspace, '/xilinx/vivado/versions') }}
```
Comment (ajelinski) [inline .github/workflows/build-packages.yml:437]: Undo this change.

File: .github/workflows/build-packages.yml [hunk 2]
```diff
@@ -676,7 +689,7 @@ jobs:
   #47
   nextpnr-ice40-windows:
     runs-on: "windows-latest"
-    needs: ["icestorm-windows"] 
+    needs: ["icestorm-windows"]
```
Comment (ajelinski) [inline .github/workflows/build-packages.yml:692]: Undo this change.

**Review by ajelinski — COMMENTED** [review]
Try not to modify anything else than necessary to make the change stated by the commit. Add a new commit that fixes them explicitly if you see anything that should be changed.

**Comment by mtdudek** [comment]
Removed unnecessary changes

**Review by ajelinski — APPROVED** [review]
